packages/ovm-toolchain/test/test-waffle-v2/core/create2.spec.ts

```diff
@@ -1,15 +1,17 @@
-import './setup'
+import '../../common/setup'
 
 /* External Imports */
 import { add0x } from '@eth-optimism/core-utils'
-import { createMockProvider, deployContract, getWallets } from 'ethereum-waffle'
-import { addHandlerToProvider } from '@eth-optimism/rollup-full-node'
-import { Contract, Wallet } from 'ethers'
-import { getAddress, keccak256, solidityPack } from 'ethers/utils'
+import { deployContract } from 'ethereum-waffle-v2'
+import { Contract, Wallet } from 'ethers-v4'
+import { getAddress, keccak256 } from 'ethers-v4/utils'
+
+/* Internal Imports */
+import { waffleV2 } from '../../../src/waffle/waffle-v2'
```

Comment:
How will our users import this? Are we going to make an `@eth-optimism/waffleV2` package?

Reply:
You can `import { waffleV2 } from '@eth-optimism/ovm-toolchain'` although I'm also considering moving to something like `import { MockProvider } from '@eth-optimism/ovm-toolchain/waffle-v2'`. I think it's a bit cleaner.